VIDEO_4_SCRIPT.md

```diff
@@ -0,0 +1,58 @@
+# Video 4: AsyncAPI and OpenAPI
+
+Hello again! Welcome to a new video about AsyncAPI, the open source initiative that seeks to improve the current state of Event Driven Architectures (EDA). If you wanna know more about the spec just check the previous video (3) on "What is AsyncAPI". This video will discuss what AsyncAPI and OpenAPI have in common and how they differ. 
+
+Ready or not, here we go! 
+
+## What is OpenAPI
+
+OpenAPI is a standard for describing application programming interfaces (APIs). This specification defines an open, vendor-independent description format for API services. In particular, OpenAPI can describe, develop, test, and document REST-compliant APIs.
```

Comment:
```suggestion
OpenAPI is a standard for describing APIs based on HTTP protocol. This specification defines an open, vendor-independent description format for API services. In particular, OpenAPI can describe, develop, test, and document REST-compliant APIs.
```

API was introduces in earlier videos, we do not need to say application programming interface again